SampleViewer/SampleManager.cpp

```diff
@@ -397,12 +389,8 @@ void SampleManager::setApiKey(bool isSupportsApiKey)
     qWarning() << "This sample expects an API key to be set, but none was provided. Please provide an API key in SampleViewer/SampleManager.cpp";
   }
   const QString sampleApiKey = isSupportsApiKey ? apiKey : ""; // empty string will "unset" the key
-  // set apikey for CPP/QML sample viewer
-#ifdef CPP_VIEWER
+  // set apikey for the sample viewer
   ArcGISRuntimeEnvironment::setApiKey(sampleApiKey);
-#else
-  emit apiKeyRequired(sampleApiKey);
```

Comment:
For QML we used to emit that signal to set the apikey but as you can see here for C++ it stays the same.